### PR TITLE
Added Google authenticator

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | no
| Fixed tickets | #2822 
| License       | MIT

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -237,12 +237,18 @@ scheb_two_factor:
...
-        enabled: "%twofactor_auth%"
+        enabled: false
         sender_email: "%twofactor_sender%"
         digits: 6
         template: WallabagUserBundle:Authentication:form.html.twig
         mailer: wallabag_user.auth_code_mailer
 
+    google:
+        enabled: true
+        server_name: wallabag
```
Comment (tcitworld) [inline app/config/config.yml]: One of these should depend on the instance's domain.

Comment (tcitworld) [inline app/config/config.yml]: Issuer can be wallabag but server_name should be a new `%domain_name%`.

Comment (tcitworld) [inline app/config/config.yml]: See https://github.com/wallabag/wallabag/pull/3173

Comment (j0k3r) [inline app/config/config.yml]: Regarding #3173 I was wondering if the Google auth needs a _domain name_ with or without the protocol?

Comment (tcitworld) [inline app/config/config.yml]: AFAIK it doesn't care about that, it's just an information to generate crypto.

**Comment by tcitworld** [comment]
Maybe refer to it as OTP instead of Google Auth.

**Comment by childnode** [comment]
@nicosomb can you clarify why this has been closed / aborted?

**Pull request closed without merging**